django_project/ta_system/admin.py

```diff
@@ -19,8 +23,9 @@ def get_urls(self):
         urls = super().get_urls()
         urls = [
             path('', self.admin_view(self.index)),
-            path('course_data_upload', self.admin_view(self.course_data_upload), name='course_data_upload')
-            ] + urls
+            path('course_data_upload', self.admin_view(
+                self.course_data_upload), name='course_data_upload')
```

Comment:
If we are going to reformat this line, let's do it like the following image where we break the line after each argument, rather than breaking the line in the middle of the second argument as you have here.

```suggestion
            path('course_data_upload',
                 self.admin_view(self.course_data_upload),
                 name='course_data_upload')
```